code/modules/modular_computers/hardware/gps_sensor.dm

```diff
@@ -11,6 +11,7 @@
 	var/datum/gps_data/gps
 
 /obj/item/weapon/computer_hardware/gps_sensor/New()
+	..()
```

Comment:
Shouldn’t that be initialize?

Reply:
No difference

Reply:
Acutually difference is exist. `Initialize` not affect gamestart lag and more controlled.

Reply:
Ghost is correct. You should always use Initialize unless you have a specific reason to use New
That doesn't apply to datums since they don't have initialize, but apply it for all atoms (objects, turfs, mobs)

Reasons to use it do exist, but it should